pkgs/applications/terminal-emulators/tabby/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchurl, pkgs, lib }:
+
+stdenv.mkDerivation rec {
+  name = "tabby";
+  version = "1.0.207";
+  src = fetchurl {
+    url = "https://github.com/Eugeny/tabby/releases/download/v${version}/tabby-${version}-linux-x64.deb";
+    hash = "sha256-PTTkL3+mYb7KM8fDUmgCuAF2lU88fYOstGWp/O5WZas=";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    dpkg
+    makeWrapper
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    dpkg -x $src .
+
+    runHook postUnpack
+  '';
+
+  buildInputs = with pkgs; [
+    glib
+    glibc
+    nss
+    nspr
+    atk
+    cups
+    dbus
+    libdrm
+    gtk3
+    pango
+    cairo
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    libxkbcommon
+    mesa
+    expat
+    xorg.libxcb
+    alsaLib
+    libGL
+    libsecret
+    musl
```

Comment:
glibc and musl required at the same time? That can't be right...

Reply:
I removed `glibc` and it still works, so `musl` is the only one needed out of the two